src/global/helpers/focus.ts

```diff
@@ -29,6 +31,30 @@ export function getFocusableElements(
   return [...focusableEls];
 }
 
+// Set the input modality in order to avoid showing the outline in Safari.
+export function seModalityOnNextFocus(elementToFocus: HTMLElement): void {
```

Comment:
maybe this method would better fit to the input-modality file?

Reply:
Ah yes, I forgot to ask the same question. I'm not sure though, would you move it there? And inside or outside the class?

Reply:
Agree